installbuilder/datafiles/Base_DSC.data

```diff
@@ -67,7 +67,7 @@ SHLIB_EXT: 'so'
 /opt/microsoft/omsconfig/module_packages/nxFileInventory_1.3.zip; release/nxFileInventory_1.3.zip; 755; ${{RUN_AS_USER}}; root
 
 #if PFARCH == x86
-/opt/microsoft/omsconfig/module_packages/nxOMSAgentNPMConfig_2.1.zip; release/nxOMSAgentNPMConfig_2.1.zip; 755; ${{RUN_AS_USER}}; root
+/opt/microsoft/omsconfig/module_packages/nxOMSAgentNPMConfig_2.2.zip; release/nxOMSAgentNPMConfig_2.2.zip; 755; ${{RUN_AS_USER}}; root
```

Comment:
why is the version changing for NPM?

Reply:
someone in NPM team bumped that version, and caused x86 build to fail